WordPress/src/main/java/org/wordpress/android/ui/posts/PostUploadService.java

```diff
@@ -775,14 +798,17 @@ private String uploadImageFile(Map<String, Object> pictureParams, MediaFile mf,
         }
 
         private Object uploadFileHelper(Object[] params, final File tempFile) {
+            AppLog.w(T.POSTS, "uploadFileHelper: " + params);
```

Comment:
AS detects this as a probable bug ("Implicit call to 'toString()' on array params"), and suggests using `Arrays.toString(params)` instead.